Добавил имя и фамилию.

Ivan-Parkhomenka opened the pull request:
Выполнил пункт 1. (Проверка взаимодействия)

Ivan-Parkhomenka committed:
Update README.md

Ivan-Parkhomenka committed:
Сделал разметку главной страницы

Сделал разметку главной страницы, но забыл про БЭМ, как переделаю сброшу верный вариант.
Везде где нет тега <img/> там будет добавлен контент посредством псевдоэлементов.
Это как правило картинки не несущие особого смысла в своем роде.

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,406 @@
+            </div>
+            <input type="text" class="user-search" placeholder="Поиск">
+            <div class="user-contact">
+                <span>+7 (495) 123-45-67</span>
```
наверное лучше сделать что нить такое `<a href="tel:+1234567890"> +7 (495) 123-45-67 </a>`, с мобилки номер будет сразу набран и роботы будут индексировать это как телефон, а с десктопа можно перенаправить вызов на мобилу

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,406 @@
+                <li><a href="#">Покупателям</a></li>
+                <li>
+                    <a href="#">О компании</a>
+                    <button class="show-about"></button>
```
пустая кнопка?? точно нужна???

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,406 @@
+                        торговой марки со скидкой 10%. 
+                        Не упустите эту возможность!</p>
+                    <a href="" class="show-catalog">Перейти в каталог</a>
+                    <ol class="slider-list">
```
если это разметка для слайдера, не думаю что тут нужен нумерованный список, порядок слайдов не имеет значения

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,406 @@
+                        <span class="code-good">Артикул: 00674157</span>
+                        <p class="price-good">3 100 руб.</p>
+                        <p class="color-good">Цвет:</p>
+                        <ol class="color-good-items">
```
здесь тоже скорее подойдет обычный список

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,406 @@
+    <footer class="main-footer">
+        <ul>
+            <li>
+                <div class="p"><span class="bold-text">Сотбит Оригами</span> – универсальный шаблон 
```
`<p>` в `<div>` с классом 'p' зачем?

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,406 @@
+                <p class="phone-sotbit">
+                    +7 (495) 111-11-11 
+                    <button class="another-phones-sotbit"></button>
+                    <a href="#"></a>
```
кнопка + ссылка..... непонятно, сможешь объяснить зачем, тогда соглашусь

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,406 @@
+                <h4>Подписаться на нас</h4>
+                <form action="URL" class="subscribe-form-footer">
+                    <input type="text" class="subscribe-input" placeholder="Ваш e-mail">
+                    <button class="subscribe-footer-btn"></button>
```
наверное этой кнопке нужен type submit, нет?????

sotbit reviewed (commented):
исправляй ошибки, и комить

Ivan-Parkhomenka commented:
Это старый файл, не тот который я тебе показывал.
Мне захотелось переписать все, чтоб по красоте было, тут все криво
Я сброшу другой, нормальный.

пт, 19 июн. 2020 г. в 09:55, sotbit <notifications@github.com>:

> *@sotbit* commented on this pull request.
>
> исправляй ошибки, и комить
> ------------------------------
>
> In source/Home_1344.html
> <https://github.com/sotbit/-learnFront-Parchomenko/pull/1#discussion_r442654111>
> :
>
> > +    <title>Origami</title>
> +</head>
> +<body>
> +    <!--Если нет тега img, значит иконки будут подгруженны посредством псевдоэлементов before и after-->
> +    <!--HEADER-->
> +    <header class="main-header">
> +
> +        <div class="header-opportunities">
> +            <img src="img" alt="" class="logo">
> +            <div class="user-placement">
> +                <span>Ваш город</span>
> +                <button class="placement-btn">Санкт-Петербург</button>
> +            </div>
> +            <input type="text" class="user-search" placeholder="Поиск">
> +            <div class="user-contact">
> +                <span>+7 (495) 123-45-67</span>
>
> наверное лучше сделать что нить такое <a href="tel:+1234567890"> +7 (495)
> 123-45-67 </a>, с мобилки номер будет сразу набран и роботы будут
> индексировать это как телефон, а с десктопа можно перенаправить вызов на
> мобилу
> ------------------------------
>
> In source/Home_1344.html
> <https://github.com/sotbit/-learnFront-Parchomenko/pull/1#discussion_r442654793>
> :
>
> > +        </div>
> +        <nav class="header-navigation">
> +            <ul>
> +                <li>
> +                    <a href="" class="nav-categories"></a>
> +                    <a href="#">Каталог</a>
> +                    <button class="show-categories"></button>
> +                </li>
> +                <li><a href="#">Акции</a></li>
> +                <li><a href="#">Бренды</a></li>
> +                <li><a href="#">Оплата</a></li>
> +                <li><a href="#">Доставка</a></li>
> +                <li><a href="#">Покупателям</a></li>
> +                <li>
> +                    <a href="#">О компании</a>
> +                    <button class="show-about"></button>
>
> пустая кнопка?? точно нужна???
> ------------------------------
>
> In source/Home_1344.html
> <https://github.com/sotbit/-learnFront-Parchomenko/pull/1#discussion_r442655512>
> :
>
> > +    <!--MAIN CONTENT-->
> +    <main class="main-container">
> +
> +        <!--PROMO SECTION-->
> +        <section class="promo-section">
> +            <h1 class="visualy-hiden">Заголовок</h1>
> +            <div class="promo-advertising">
> +                <div class="promo-slider">
> +                    <span>Акции</span>
> +                    <h2>Скидки на все товары бренда "Nike"</h2>
> +                    <p>Мы предлагаем вам стать частью большой семьи Levi’s:
> +                        до 31 декабря вы можете купить любую одежду этой
> +                        торговой марки со скидкой 10%.
> +                        Не упустите эту возможность!</p>
> +                    <a href="" class="show-catalog">Перейти в каталог</a>
> +                    <ol class="slider-list">
>
> если это разметка для слайдера, не думаю что тут нужен нумерованный
> список, порядок слайдов не имеет значения
> ------------------------------
>
> In source/Home_1344.html
> <https://github.com/sotbit/-learnFront-Parchomenko/pull/1#discussion_r442655950>
> :
>
> > +        <!--THE MOST POPULAR GOODS SECTION-->
> +        <section class="popular-goods-section">
> +            <h2 class="visualy-hiden">Заголовок</h2>
> +            <button class="discount-goods-btn">Акция</button>
> +            <button class="discount-hits-btn">Акция</button>
> +            <div class="popular-goods-items">
> +                <ul>
> +                    <li>
> +                        <img src="img" alt="">
> +                        <a href="#" class="good-link">Кеды SB DELTA FORCEVULC, Nike</a>
> +                        <input type="radio" name="stars" value="4">
> +                        <span class="quantity-good">37 шт</span>
> +                        <span class="code-good">Артикул: 00674157</span>
> +                        <p class="price-good">3 100 руб.</p>
> +                        <p class="color-good">Цвет:</p>
> +                        <ol class="color-good-items">
>
> здесь тоже скорее подойдет обычный список
> ------------------------------
>
> In source/Home_1344.html
> <https://github.com/sotbit/-learnFront-Parchomenko/pull/1#discussion_r442663080>
> :
>
> > +            <p>Оставьте нам свои данные и наши менеджеры свяжутся с вами!</p>
> +            <form action="URL" class="main-form">
> +                <input type="text" placeholder="Ваше имя">
> +                <input type="text" placeholder="E-mail *">
> +                <input type="text" placeholder="Номер телефона *">
> +                <button class="get-in-touch-us-btn">Связаться с нами</button>
> +                <input type="checkbox" class="agreement-form">
> +            </form>
> +        </section>
> +    </main>
> +
> +    <!--FOOTER-->
> +    <footer class="main-footer">
> +        <ul>
> +            <li>
> +                <div class="p"><span class="bold-text">Сотбит Оригами</span> – универсальный шаблон
>
> <p> в <div> с классом 'p' зачем?
> ------------------------------
>
> In source/Home_1344.html
> <https://github.com/sotbit/-learnFront-Parchomenko/pull/1#discussion_r442663455>
> :
>
> > +    </main>
> +
> +    <!--FOOTER-->
> +    <footer class="main-footer">
> +        <ul>
> +            <li>
> +                <div class="p"><span class="bold-text">Сотбит Оригами</span> – универсальный шаблон
> +                    для создания интернет-магазинов любой тематики.
> +                    Многие знакомы с принципом оригами – за основу берется квадратный лист бумаги,
> +                    из которого складывается фигурка.  </div>
> +                <p class="placement-sotbit">ул. Ленина, 15</p>
> +                <p class="mail-sotbit">sales@origami.sotbit.ru</p>
> +                <p class="phone-sotbit">
> +                    +7 (495) 111-11-11
> +                    <button class="another-phones-sotbit"></button>
> +                    <a href="#"></a>
>
> кнопка + ссылка..... непонятно, сможешь объяснить зачем, тогда соглашусь
> ------------------------------
>
> In source/Home_1344.html
> <https://github.com/sotbit/-learnFront-Parchomenko/pull/1#discussion_r442663818>
> :
>
> > +                <h4>Помощь</h4>
> +                <ul>
> +                    <li><a href="#">Оплата</a></li>
> +                    <li><a href="#">Возврат товара</a></li>
> +                    <li><a href="#">Доставка</a></li>
> +                    <li><a href="#">Как оформить заказ</a></li>
> +                    <li><a href="#">Правила продажи товаров</a></li>
> +                    <li><a href="#">Публичная оферта</a></li>
> +                    <li><a href="#">Конфиденциальность</a></li>
> +                </ul>
> +            </li>
> +            <li>
> +                <h4>Подписаться на нас</h4>
> +                <form action="URL" class="subscribe-form-footer">
> +                    <input type="text" class="subscribe-input" placeholder="Ваш e-mail">
> +                    <button class="subscribe-footer-btn"></button>
>
> наверное этой кнопке нужен type submit, нет?????
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sotbit/-learnFront-Parchomenko/pull/1#pullrequestreview-433830942>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APBB5FJPBDJXD3QXPG2LED3RXMDVTANCNFSM4M5WHFBQ>
> .
>

Ivan-Parkhomenka committed:
Разметил 01_home_1344

Разметил согласно семантике, главную страницу макета "Оригами".
Прошу максимально деструктивной критики, чтобы был рост.

Ivan-Parkhomenka commented:
Разметил 01_home_1344, прошу проверки.

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,639 @@
+            </div>
+            <input type="text" class="header-inner__search" placeholder="Поиск">
+            <div class="header-inner__location">
+                <button class="header-inner__phone-number">+7 (495) 123-45-67</button>
```
смотри комментарии в предыдущем коммите

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,639 @@
+            <ul class="header-inner__goods-menu">
+                <li>
+                    <a href="#">
+                        <img src="" alt="График" class="header-inner__goods-img">
```
подсказка, лучше делать это svg, потом когда будет меняться цвет не надо будет собирать спрайт png

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,639 @@
+                    Не упустите эту возможность!
+                </p>
+                <a href="#" class="preview-slider__link">Перейти в каталог</a>
+                <div class="preview-slider__items">
```
наверное лучше это будет сделать кнопками (они будут доступны с клавиатуры)

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,639 @@
+                <li class="popular-menu__item">
+                    <a href="#" class="popular-menu__link">
+                        <img src="" alt="Бытовая техника" class="popular-menu__img">
+                        <sapm class="popular-menu__link-text">Бытовая техника</sapm>
```
<samp> ???? зачем??

Ivan-Parkhomenka replied:
Это ведь блоки с ссылками на элементы каталога поэтому обернул их в <a></a>

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,639 @@
+                    </a> 
+                </li>
+            </ul>
+            <div class="popular-goods">
```
мне кажется, что популярные категории  и карточки товаров это структурно разные блоки

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,639 @@
+                                    <span class="popular-goods__color-value popular-goods__color-value--grn popular-goods__color-value--checked"></span>
+                                </li>
+                                <li class="popular-goods__color-item">
+                                    <span class="popular-goods__color-value popular-goods__color-value--gry"></span>
```
по идее тут не совсем верно вешать модификаторы, смотри.....если цветов будет 50? будет 50 модификаторов?, а если цвет бело-черно-желтый-серо-буро-малиновый? представляю модификатор, но это больше придирка с расчетом на использование (на данном этапе такая задача не стоит, поэтому можешь не исправлять)

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,639 @@
+                        <form action="" class="popular-goods__item-rating" method="POST">
+                            <label for="value-one" class="popular-goods__rating-ico">
+                                <input type="radio" name="rating" value="1" class="popular-goods__rating-star" id="value-one">
+                            </label>
```
`type="radio"` для рейтинга не совсем подходит, как ты выберешь сразу 4 звезды?

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,639 @@
+                            <ul class="popular-goods__color-list">
+                                <li class="popular-goods__color-item">
+                                    <span class="popular-goods__color-value popular-goods__color-value--grn popular-goods__color-value--checked"></span>
+                                </li>
```
скорее всего это будет type="radio", тебе же выбранный цвет нужно отправить, и он будет только один

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,639 @@
+                                <li class="popular-goods__size-item">
+                                    <span class="popular-goods__size-value  popular-goods__size-value--checked">39</span>
+                                </li>
+                                <li class="popular-goods__size-item">
```
аналогично, это будет форма

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,639 @@
+                    берется квадратный лист бумаги, из которого складывается фигурка.  
+                </p>
+                <p class="footer__info-location">ул. Ленина, 15</p>
+                <p class="footer__info-mail">sales@origami.sotbit.ru</p>
```
см в первом коммите

sotbit commented on source/Home_1344.html:
```diff
@@ -0,0 +1,639 @@
+                </p>
+                <p class="footer__info-location">ул. Ленина, 15</p>
+                <p class="footer__info-mail">sales@origami.sotbit.ru</p>
+                <button class="footer__info-phone">+7 (495) 111-11-11</button>
```
тоже самое

Ivan-Parkhomenka committed:
Разметил три страницы

Разметил страницы:
 -Блог;
 -Главная;
 -Детально;

sotbit commented on source/detaild.html:
```diff
@@ -0,0 +1,477 @@
+                            </span>
+                        </p>
+                        <div  class="prices__good__incr">
+                           <span class="prices__good__incr__btn">-</span>
```
кнопка

sotbit commented on source/detaild.html:
```diff
@@ -0,0 +1,477 @@
+                        </p>
+                        <div  class="prices__good__incr">
+                           <span class="prices__good__incr__btn">-</span>
+                           <span class="prices__good__incr__value">1</span>
```
инпут

sotbit commented on source/detaild.html:
```diff
@@ -0,0 +1,477 @@
+                        <div  class="prices__good__incr">
+                           <span class="prices__good__incr__btn">-</span>
+                           <span class="prices__good__incr__value">1</span>
+                           <span class="prices__good__incr__btn">+</span>
```
кнопка

sotbit commented on source/detaild.html:
```diff
@@ -0,0 +1,477 @@
+                </p>
+                <p class="footer__info-location">ул. Ленина, 15</p>
+                <p class="footer__info-mail">sales@origami.sotbit.ru</p>
+                <button class="footer__info-phone">+7 (495) 111-11-11</button>
```
ссылка tel

sotbit commented on source/detaild.html:
```diff
@@ -0,0 +1,477 @@
+                    <span class="good-inner__img__text-blue">Новинка</span>
+                    <li class="good-inner__img__item">
+                        <a href="№" class="good-inner__img__link">
+                            <img src="№" alt="Смартфон" class="good-inner__img_">
```
что за класс заканчивающийся на _
